styles/settings-view.less

```diff
@@ -698,6 +698,14 @@
   }
 }
 
+.package-updates-status-view
```

Comment:
There is an opening `{` missing at the end.

We could also simplify the selector to:

```css
.package-updates-status-view:hover {
  text-decoration: underline;
}
```

The icon wouldn't get the underline.